user-service/src/main/java/hcmus/alumni/userservice/dto/ISearchDto.java

```diff
@@ -0,0 +1,9 @@
+package hcmus.alumni.userservice.dto;
+
+public interface ISearchDto {
```

Comment:
```suggestion
public interface UserSearchDto {
```
trả về thêm avatar, roles của ng đó